algebird-test/src/main/scala/com/twitter/algebird/scalacheck/PosNum.scala

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2016 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.twitter.algebird
+package scalacheck
+
+import org.scalacheck.{ Arbitrary, Gen }
+
+case class PosNum[T: Numeric](value: T)
```

Comment:
Removed! We didn't need it at construction. I wanted to enforce that it was a numeric type, but the arb does that.